### ISSUE TITLE
Uncaught exception when trying to sort the menu

### ISSUE THREAD

**Issue by Majea** [otigon/automated-jb2a-animations#386]
Since the 3.0.0 release, the "sort menu" button doesn't work anymore. The following stack trace is generated: 
```
auto-recognition.js:473 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
[Detected 1 package: autoanimations]
    at Function.keys (<anonymous>)
    at aaAutoRecognition.sortMenu (auto-recognition.js:473:29)
    at aaAutoRecognition.sortAutorec (auto-recognition.js:466:38)
    at async HTMLDivElement.<anonymous> (auto-recognition.js:143:13)
```

I'm now using Automated Animation 3.0.3 with Foundry 9.266

**Comment by otigon** [comment]
Thanks for the report, I think I know what's causing that and will have a fix in the next release. In the meantime, if you are on DnD 5e you can navigate to the "Active Effects" tab of the Autorec menu and create a dummy section

**Comment by otigon** [comment]
Issue is fixed as of 3.0.3

**Comment by Majea** [comment]
Thanks for the fix!
